pom.xml

```diff
@@ -126,21 +125,25 @@
         <maven.compiler.target>${java.version}</maven.compiler.target>
 
 		<log4j.version>1.2.16</log4j.version>
-		<spring.version>4.3.19.RELEASE</spring.version>
+		<spring.version>5.1.7.RELEASE</spring.version>
 		<spring.groupId>org.springframework</spring.groupId>
-		<slf4j.version>1.7.10</slf4j.version>
-		<gigaspaces.version>14.0.1</gigaspaces.version>
+		<slf4j.version>1.7.26</slf4j.version>
+		<gigaspaces.version>14.5.0</gigaspaces.version>
 		<rxjava.version>1.3.8</rxjava.version>
 		<hystrix.version>1.5.18</hystrix.version>
 		<archaius.version>0.4.1</archaius.version>
-		<jackson2.version>2.8.11</jackson2.version>
-		<guava.version>20.0</guava.version>
-		<junit.version>4.11</junit.version>
-		<hamcrest.version>1.2.1</hamcrest.version>
-		<mockito.version>1.9.0</mockito.version>
+		<jackson2.version>2.11.2</jackson2.version>
+		<guava.version>29.0-jre</guava.version>
+		<junit.version>4.12</junit.version>
+		<hamcrest.version>1.3</hamcrest.version>
+		<mockito.version>2.23.0</mockito.version>
 		<lifecycle-mapping.version>1.0.0</lifecycle-mapping.version>
 		<dropwizard.version>3.1.2</dropwizard.version>
-		<netty.version>4.0.31.Final</netty.version>
+		<netty.version>4.1.30.Final</netty.version>
+		<mimer-config.version>0.0.4</mimer-config.version>
+		<hystrix-multiconfig.version>0.0.3</hystrix-multiconfig.version>
+		<gs-test.groupId>com.avanza.gs14</gs-test.groupId>
+		<gs-test.version>14.5.0-SNAPSHOT</gs-test.version>
```

Comment:
This version needs to be updated once https://github.com/AvanzaBank/gs-test/pull/21 has been merged, built and released, before this PR can be merged.